adapters/adform/params_test.go

```diff
@@ -66,4 +70,8 @@ var invalidParams = []string{
 	`{"mid":"123","mkv":"color:blue,l&ngth:350"}`,
 	`{"mid":"123","mkv":"color::blue"}`,
 	`{"mid":"123","mkw":"fem&le"}`,
+	`{"mid":"123","minp":"2.1"}`,
+	`{"mid":"123","cdims":"500x300:400:200","mkw":" "}`,
```

Comment:
The colon (:) separator doesn't work and instead a comma (,) should be used so: _"cdims":"500x300:400:200"_ should become _"cdims":"500x300,400,200"_